LaTeX/LaTeX.sublime-syntax

```diff
@@ -634,16 +634,32 @@ contexts:
         4: variable.parameter.function.latex
         5: punctuation.definition.group.brace.end.tex
       push: block-math-begin-end-command-body
+    - match: |-
+        (?x)
+        ((\\)begin)
+        \s* (\{) \s* ( IEEEeqnarray\*? ) \s* (\})
+        \s* (\{) \s* ( {{letter}}+ ) \s* (\})
```

Comment:
What is the motivation behind this change? Afaics this marks any begin block not matched otherwise as a math block and that seems a bit over-eager, does it not?

Reply:
Motivation is support for `\begin{IEEEeqnarray}{c}`

1. `\s* (\{) \s* ( IEEEeqnarray\*? ) \s* (\})` matches `{IEEEeqnarray}`
2. `\s* (\{) \s* ( {{letter}}+ ) \s* (\})` matches `{c}`

Actually, 2. should probably be optional.

Reply:
Btw.: we need to consume that as otherwise LaTeXTools includes it into math equation preview.

Reply:
>  Afaics this marks any begin block not matched otherwise as a math block

Nope.

Reply:
Ah, I see. I somehow inserted a `|` between these two in my head and did not consider it to be the second parameter.